### PR TITLE
feat(store): add xbox series x and s

### LINKED ISSUE #506: Add Xbox Series X/S and PS5
### Description

Would be cool if we could add Xbox Series X/S and PS5 to this project so that it retains it's usefulness as more people get the RTX 3080.

### Possible solution

Just add the URLs, etc.

### PR DESCRIPTION
**Pull request by vnguyen00**
### Description

Closes #506

Adding Xbox Series X and S to the scanner

I am trying to purchase an Xbox on release day :)

### Testing
Ran npm start run locally and scanner passed back expected results.

went through lint

npm start run

modified models in .env file to search for xboxsx and xboxss

### CODE REVIEW

**Comment by vnguyen00** [comment]
The Walmart XBX page will error out for now. The page is non-functional for now. It should keep the same URI
[11:20:54 AM] warn :: ✖ [walmart] [microsoft (xboxsx)] xbox series x :: STATUS CODE ERROR 404

**Comment by jef** [comment]
Overall, looks good to me!

> The Walmart XBX page will error out for now. The page is non-functional for now. It should keep the same URI
> [11:20:54 AM] warn :: ✖ [walmart] [microsoft (xboxsx)] xbox series x :: STATUS CODE ERROR 404

I don't mind this error for now since it'll be stated; but how did you end up getting these IDs?

**Comment by vnguyen00** [comment]
> I don't mind this error for now since it'll be stated; but how did you end up getting these IDs?

From this page, https://www.walmart.com/cp/xbox-series/9206773 .
The series S link works just not the X.

**Comment by mven** [comment]
Don't forget about the additions to the readme.

File: src/store/model/amazon.ts [hunk 1]
```diff
@@ -273,6 +273,18 @@ export const Amazon: Store = {
 			model: 'ps5 digital',
 			series: 'sonyps5de',
 			url: 'https://www.amazon.com/dp/B08FC6MR62'
+		},
+		{
+			brand: 'microsoft',
+			model: 'xbox series x',
+			series: 'xboxsx',
+			url: 'https://www.amazon.com/Xbox-X/dp/B08H75RTZ8/ref=sr_1_1?crid=2LDCJ421GASAD&dchild=1&keywords=xbox+series+x&qid=1604939295&sprefix=xbox+seri%2Caps%2C245&sr=8-1'
```
Comment (v1ct0rv) [inline src/store/model/amazon.ts:281]:
```suggestion
			url: 'https://www.amazon.com/dp/B08H75RTZ8'
```

File: src/store/model/amazon.ts [hunk 2]
```diff
@@ -273,6 +273,18 @@ export const Amazon: Store = {
...
+		},
+		{
+			brand: 'microsoft',
+			model: 'xbox series s',
+			series: 'xboxss',
+			url: 'https://www.amazon.com/Xbox-X/dp/B08G9J44ZN/ref=sr_1_1?crid=2LDCJ421GASAD&dchild=1&keywords=xbox%2Bseries%2Bx&qid=1604939295&sprefix=xbox%2Bseri%2Caps%2C245&sr=8-1&th=1'
```
Comment (v1ct0rv) [inline src/store/model/amazon.ts:287]:
```suggestion
			url: 'https://www.amazon.com/dp/B08G9J44ZN'
```

**Comment by v1ct0rv** [comment]
> Don't forget about the additions to the readme.

Yes, the README is very important.

**Comment by vnguyen00** [comment]
> Don't forget about the additions to the readme.

Done. I added to supported 

> > Don't forget about the additions to the readme.
> 
> Yes, the README is very important.

Done! Added to the brands and models section. LMK if there's anything else.

**Review by jef — APPROVED** [review]
Thanks!

**Pull request merged**